### ISSUE TITLE
ke_rho is a truncated-moment ratio, but the docs state it as the full-distribution one

### ISSUE THREAD

**Issue by duncanwp** [reflective-org/neuralmie-jax#4]
`reference/bulk_optics.py` integrates **both** the numerator and the denominator over the interval that brackets `INTFRAC = 0.999` of the **number** distribution. That is faithful to upstream and to the paper, and it is what the networks were trained against — the issue is that nothing says so where a consumer will read it. The README and the module docstring both give the target as

```
ke_rho = 0.75 * INT(Qe*pdf*r^2 dlnr) / INT(pdf*r^3 dlnr)
```

with no bounds, i.e. as the *full*-distribution moment ratio. It isn't one.

### Why the truncation does not cancel

In log-radius the number density is normal with mean `ln(mu)` and s.d. `s = ln(sigma_g)`. The `r^3` weight in the denominator shifts that density to mean `3s`; the `r^2` weight in the numerator shifts it to `2s`. The integration window is symmetric about `ln(mu)` at `+/- z*s` with `z = Phi^-1(0.9995) = 3.29053`, so the two integrals lose *different* fractions of their mass:

| | retained fraction at `sigma_g = 1.8` |
|---|---|
| number | 99.900 % (by construction) |
| numerator (`r^2`-weighted) | `Phi(z-2s) - Phi(-z-2s)` = 98.278 % |
| denominator (`r^3`-weighted) | `Phi(z-3s) - Phi(-z-3s)` = **93.664 %** |

So the ratio is biased even though 99.9 % of the particles are inside the window.

### Measured, using only this repository

```python
from neuralmie_jax.reference.bulk_optics import bulk_optics_reference
lam, rg, mr, mi = 5.5e-7, 1.0e-7, 1.53, 5e-3
for sg in (1.4, 1.6, 1.8, 2.0):
    t = bulk_optics_reference(lam, rg, sg, mr, mi)[0]                    # intfrac=0.999
    f = bulk_optics_reference(lam, rg, sg, mr, mi, intfrac=1 - 1e-9)[0]  # effectively full
    print(sg, t, f, t / f - 1)
```

| `sigma_g` | `ke_rho` truncated (1/m) | `ke_rho` full (1/m) | truncated/full − 1 |
|---|---:|---:|---:|
| 1.4 | 5.9415e6 | 5.97824e6 | **−0.61 %** |
| 1.6 | 7.57408e6 | 7.44702e6 | **+1.71 %** |
| 1.8 | 7.41901e6 | 7.08353e6 | **+4.74 %** |
| 2.0 | 6.2094e6 | 5.65519e6 | **+9.80 %** |

No emulator and no external model is involved — this is the reference against itself, so it is a property of the training target, not of the networks or of any port.

### Why it bites a consumer

A modal aerosol scheme relates mass, number and `r_g` through the **full** lognormal third moment (`V = N (pi/6) Dg^3 exp(4.5 ln^2 sigma)`), so the natural coupling is `tau = ke_rho * V_full * (column factor)`. That pairs a truncated ratio with an untruncated volume, and the mismatch above survives in full. It is a few percent on optical depth across the `sigma_g` range modal schemes actually use — comparable to, or larger than, the emulator's own quoted accuracy (0.18 % max), which is measured against this same truncated reference and therefore cannot see it.

Note the sign flips between `sigma_g` 1.4 and 1.6, so a constant volume-fraction correction is not a fix: the numerator and denominator deficits cross over.

### Suggested

1. State the convention in the README's units table and in the `bulk_optics` / `neuralmie` docstrings — that `ke_rho` is a ratio of integrals truncated to the 99.9 %-of-number interval, and that a consumer pairing it with a full third moment inherits the difference.
2. Quantify it as a function of `sigma_g` (the table above, extended), so a consumer can decide whether to correct or to accept it.
3. Consider adding the full-distribution figure to the accuracy sweep as a separate column — "error against the training target" and "error against full modal optics" are different numbers and both are worth publishing.

Originally observed in climate-analytics-lab/jax-gcm#794 while reviewing a downstream integration; reproduced here analytically and against this repository's own reference.

**Comment by duncanwp** [comment]
**Correction to the issue above: I understated this, and my explanation was only half right.**

I claimed the bias follows from geometry alone — that the `r^2`-weighted numerator and the `r^3`-weighted denominator lose different fractions of a symmetric window, giving `Phi(z-2s)-Phi(-z-2s)` over `Phi(z-3s)-Phi(-z-3s)`. That derivation holds `Qe` constant over the retained range. It is a good approximation only where `Qe` is slowly varying, i.e. at size parameter above ~1, which is the only regime my table sampled (`r_g` = 100 nm at 550 nm is `mu_x` = 1.14).

Below that it fails, and in the opposite direction. Where `Qe ~ x^4`, the numerator integrand is effectively `r^6`-weighted, so truncation removes far more of the numerator than of the `r^3` denominator and the bias goes strongly **negative**:

| `mu_x` (σ_g = 1.8) | 0.057 | 0.114 | 0.343 | 0.685 | 1.14 | 2.29 | 5.71 | 11.4 | 22.8 |
|---|---:|---:|---:|---:|---:|---:|---:|---:|---:|
| truncated/full − 1 | −18.97 % | **−37.32 %** | −14.15 % | +2.36 % | +4.74 % | +5.30 % | +5.17 % | +5.07 % | +5.02 % |

It asymptotes to the geometry-only value (+4.93 % at σ_g = 1.8) once `Qe` flattens, which is why the original table looked consistent with it.

```python
from neuralmie_jax.reference.bulk_optics import bulk_optics_reference
lam, sg, mr, mi = 5.5e-7, 1.8, 1.53, 5e-3
for rg in (1e-8, 3e-8, 1e-7, 1e-6):
    t = bulk_optics_reference(lam, rg, sg, mr, mi)[0]
    f = bulk_optics_reference(lam, rg, sg, mr, mi, intfrac=1 - 1e-9)[0]
    print(2 * 3.141592653589793 * rg / lam, t / f - 1)
```

The `intfrac = 1 - 1e-9` reference is converged at the default `nrad = 1024`: against `nrad = 4096` it agrees to 6e-10 / 1.8e-7 / 2.3e-5 at `mu_x` = 0.34 / 1.14 / 11.4.

Three consequences for the issue as filed:

1. **The range is about −37 % to +10 %**, not −0.6 % to +9.8 %. The worst case is small size parameter, not large `sigma_g`.
2. **This is squarely in the networks' domain, not the Rayleigh arm's.** At σ_g = 1.8 the analytic switch sits at `mu_x` ≈ 0.0145, so every row above is network-evaluated. (The Rayleigh arm's `_lognormal_moment_ratio` uses the same `SQRT2_ERFINV_999` window, so it shares the convention.)
3. **It reaches the accumulation mode in the longwave.** `mu_x` ≈ 0.1 is `r_g` ≈ 9 nm at 550 nm — Aitken only — but `r_g` ≈ 160 nm at 10 µm. Measured at 10 µm with `r_g` = 500 nm: −5.2 % / −14.5 % / −15.4 % / −7.4 % at σ_g = 1.4 / 1.6 / 1.8 / 2.0. A host's longwave aerosol optics sit in the regime where this is worst.

The ask is unchanged — document the convention and quantify it — but it should be quantified against **size parameter and `sigma_g`**, not `sigma_g` alone, and the headline number is larger than I first reported. A PR follows.
